### PR TITLE
Improve readability in CSRF section of guide

### PR DESCRIPTION
**Pull request by AndyLampert**

### CODE REVIEW

**Comment by sgrif** [comment]
Can you rebase this? It has merge conflicts w/ master.

**Comment by AndyLampert** [comment]
Thanks @sgrif . I think that did it.

**Comment by sgrif** [comment]
Not quite. This PR is now two commits.

File: guides/source/security.md [hunk 1]
```diff
@@ -224,9 +224,9 @@ The HTTP protocol basically provides two main types of requests - GET and POST (
 * The interaction _changes the state_ of the resource in a way that the user would perceive (e.g., a subscription to a service), or
 * The user is _held accountable for the results_ of the interaction.
 
-If your web application is RESTful, you might be used to additional HTTP verbs, such as PATCH, PUT or DELETE. Most of today's web browsers, however do not support them - only GET and POST. Rails uses a hidden `_method` field to handle this barrier.
+If your web application is RESTful, you might be used to additional HTTP verbs, such as PATCH, PUT or DELETE. Most of today's web browsers, however, do not support them - only GET and POST. Rails uses a hidden `_method` field to handle this barrier.
 
-_POST requests can be sent automatically, too_. Here is an example for a link which displays `www.harmless.com` as destination in the browser's status bar. In fact it dynamically creates a new form that sends a POST request.
+_POST requests can be sent automatically, too_. In this example, the link www.harmless.com is shown as the destination in the browser's status bar. But it has actually dynamically creates a new form that sends a POST request.
```
Comment (jeremy) [inline guides/source/security.md]: `But it has actually dynamically creates` → created*

**Comment by AndyLampert** [comment]
Ok, phew. I think that's it! :tired_face: 

Also added your fix @jeremy - thanks for grammar the check!

**Pull request merged**